Terminal.Gui/Core/Window.cs

```diff
@@ -201,7 +201,7 @@ public override bool MouseEvent (MouseEvent mouseEvent)
 				if (dragPosition.HasValue) {
 					if (SuperView == null) {
 						Application.Top.SetNeedsDisplay (Frame);
-						Application.Top.Redraw (Bounds);
+						Application.Top.Redraw (Frame);
```

Comment:
Please add this comment

```
// Redraw the entire app window using just our Frame. Since we are 
// Application.Top, and our Frame always == our Bounds (Location is always (0,0))
// our Frame is actually view-relative (which is what Redraw takes).
Application.Top.Redraw (Frame);
```

Reply:
Done!